### PR TITLE
Order model, dao and service added

### PR DESCRIPTION
**Pull request by hpavlo**

### CODE REVIEW

**Review by nazariipertsak — CHANGES_REQUESTED** [review]

File: src/main/java/mate/academy/model/Order.java [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+import javax.persistence.OneToMany;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "orders")
+public class Order {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @OneToMany(fetch = FetchType.EAGER)
```
[the remark this answers is not in the document]
Comment (Pavlo-Gook) [reply src/main/java/mate/academy/model/Order.java:22]: I forgot to remove it after tests 😅

**Review by nazariipertsak — APPROVED** [review]
Great! 👍🏻

File: src/main/java/mate/academy/dao/impl/OrderDaoImpl.java [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+                session.close();
+            }
+        }
+    }
+
+    @Override
+    public List<Order> getByUser(User user) {
+        try (Session session = HibernateUtil.getSessionFactory().openSession()) {
+            Query<Order> query = session.createQuery(
+                    "SELECT DISTINCT o FROM Order o "
```
Comment (ChumakAn) [inline src/main/java/mate/academy/dao/impl/OrderDaoImpl.java:42]: it should work without SELECT DISTINCT o

**Review by ChumakAn — CHANGES_REQUESTED** [review]
Good! Minor fix 🔥

**Review by ChumakAn — APPROVED** [review]

**Pull request closed without merging**